ci/README.md

```diff
@@ -24,12 +24,10 @@ In addition to the environment variables specified in [Deploying the app](../REA
 | REGION                                    | What region to authenticate against                                                  |
 | PROJECT_ID                                | The ID of the GCP target project                                                     |
```

Comment:
This table of envs vars should be removed as it is a duplicate. It just needs the link to the deploying the app section of the main readme.

Reply:
Linked now instead of this table of variables, thanks!